library/templates/cbbr_submissions.yml

```diff
@@ -0,0 +1,35 @@
+dataset:
+  name: &name cbbr_submissions
+  version: "{{ version }}"
+  acl: public-read
+  source:
+    url:
+      path: library/tmp/cbbr_submissions.csv
+      subpath: ""
+    options:
+      - "AUTODETECT_TYPE=NO"
+      - "EMPTY_STRING_AS_NULL=YES"
+    geometry:
+      SRS: null
+      type: NONE
+
+  destination:
+    name: *name
+    geometry:
+      SRS: null
+      type: NONE
+    options:
+      - "OVERWRITE=YES"
+      - "PRECISION=NO"
+    fields: []
+    sql: null
+
+  info:
+    description: |
+      ###  The cbbr_submissions (Community Board Budget Requests) data consists of budget 
+      requests submitted through NYC's various community boards. Each record consists of 
+      an indiviudal budget request and the responsible agencies response to each formal 
+      budget request. This data is compiled by DCP Strategic Planning and given to DCP Data
+      Engineering team as an xlsx file which is manually uploaded and ingested via data-library.
```

Comment:
would this be a good place to note the VLOOKUP issue and how it's fixed?

Reply:
Yeah I can add it here